insuree_batch/views.py

```diff
@@ -1,42 +1,150 @@
+import glob
+import io
 import os
 from io import BytesIO
+from pathlib import Path
+import shutil
+import subprocess
+from turtle import width
 
 from django.http import FileResponse
 from django.shortcuts import get_object_or_404, render
 
 from django.core.exceptions import PermissionDenied
 from django.utils.translation import gettext as _
+from numpy import append
 import qrcode
 import qrcode.image.svg
 
 from . import services
 from .apps import InsureeBatchConfig
 from .models import InsureeBatch
 
+import os
+from PyPDF2 import PdfFileMerger, PdfFileReader
+import svg_stack as ss
+
 
 def batch_qr(request):
     if not request.user.has_perms(InsureeBatchConfig.gql_query_batch_runs_perms):
         raise PermissionDenied(_("unauthorized"))
+
     batch_id = request.GET.get("batch")
     batch = get_object_or_404(InsureeBatch, id=batch_id)
 
     factory = qrcode.image.svg.SvgImage
-    insuree_ids = []
+
+    file_name = InsureeBatchConfig.front_template_name
```

Comment:
If no template is provided, should we just use the previous version with the list of QR codes? What would be the default no template action?

Reply:
There must be a template, just like the SMS templates.